src/Env/Config/Variable/MakeTask.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Manala package.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Env\Config\Variable;
+
+/**
+ * Manala "app" and "vendor" environment variables.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+final class MakeTask implements Variable
```

Comment:
I find `MakeTask` confusing (I though it was something that Manala will execute at some point instead of a `Variable`). 
What do you think about `MakeTarget` ?